### ISSUE TITLE
Dropping pyenv in favor of python section

### ISSUE THREAD

**Issue by matchai** [matchai/spacefish#112]
Looks like we've got a bit of updating to do for the pyenv section.
This would also be a great opportunity to add testing where we don't have any at the moment. 😄 

## 

> [🐟](https://github.com/matchai/spacefish-bot) **spacefish-bot** — Pull request copied from: https://github.com/denysdovhan/spaceship-prompt/pull/431.

#### Description

Dropping `pyenv` section in favor of `python` section, `python` section has three display modes.

|Variable|Value|Meaning|
|:----:|:--:|:---|
| `SPACESHIP_PYTHON_SHOW` | always | Always display python version information if relevant files exist in directory |
| `SPACESHIP_PYTHON_SHOW` | true (default) | Display python version only inside virtual environment if relevant files exist | 
| `SPACESHIP_PYTHON_SHOW` | false | Don't display python version in prompt | 

#### Screenshot

![python version inside virtualenv](https://user-images.githubusercontent.com/10276208/48916176-48047180-eea7-11e8-9482-de176d6cf42e.png)

#### Tasks

- [x] Update docs
- [ ] Backward compatibility

Related denysdovhan/spaceship-prompt#394 
Close denysdovhan/spaceship-prompt#410

**Comment by salmanulfarzy** [comment]
Note it was merged to 4.0 branch and not to master.

**Comment by matchai** [comment]
Good call. I'll have this planned for 2.0, which will be parallel to spaceship's 4.0 👍

**Comment by CSergienko** [comment]
Hello! Does this mean that the theme will support other python version managers, like ASDF, virtualenvwrapper, etc, in the near future?

Thank you for the awesome work you've done porting this theme to fish.

**Comment by matchai** [comment]
As soon as such features are added to spaceship, we aim to add support for equivalent features in spacefish. 😄

**Comment by CSergienko** [comment]
Fair enough.
